assembl/graphql/schema.py

```diff
@@ -113,6 +115,26 @@ def resolve_root_idea(self, args, context, info):
         root_thematic = get_root_thematic_for_phase(discussion, identifier)
         return root_thematic
 
+    def resolve_vote_session(self, args, context, info):
+
+        discussion_phase_id = args.get('discussion_phase_id')
+        discussion_phase = models.DiscussionPhase.get(discussion_phase_id)
+        # TODO: see if we can avoid this next(iter( thing with a one-to-one relationship
+        vote_session = next(iter(discussion_phase.vote_session or []), None)
+        return vote_session
+
+    def resolve_vote_sessions(self, args, context, info):
+        vote_sessions = []
+        discussion_id = context.matchdict['discussion_id']
+        discussion = models.Discussion.get(discussion_id)
+        for discussion_phase in discussion.timeline_phases:
+            if discussion_phase.identifier == "tokenVote":
+                vote_session = next(iter(discussion_phase.vote_session or []), None)
+                if vote_session is None:
+                    vote_session = models.VoteSession(discussion_phase=discussion_phase)
+                vote_sessions = vote_sessions + [vote_session]
```

Comment:
`vote_sessions.append(vote_session)` is more pythonic here.

Reply:
this part of the code is broken but unused, should I fix or remove the resolve_vote_sessions altogether ? (frontend use the single vote_session resolver)

Reply:
You should remove unused code, specially if it's broken and non tested code.